packages/account/src/Sections/Profile/FinancialAssessment/financial-assessment.jsx

```diff
@@ -137,6 +137,7 @@ class FinancialAssessment extends React.Component {
     is_mounted = false;
     state = {
         is_loading: true,
+        has_completed_financial_assessment: false,
```

Comment:
can be removed from the state as it's not used

Reply:
Agreed and will be incorporated.